pkg/validation/spec/swagger_test.go

```diff
@@ -277,12 +289,6 @@ func TestSwaggerSpec_ExperimentalUnmarshal(t *testing.T) {
 	if !reflect.DeepEqual(control, actual) {
 		t.Fatal(cmp.Diff(control, actual, SwaggerDiffOptions...))
 	}
-
-	newJsonBytes, err := json.Marshal(actual)
-	require.NoError(t, err)
-	if err := jsontesting.JsonCompare(jsonBytes, newJsonBytes); err != nil {
```

Comment:
was this dropped on purpose?

Reply:
These two were dropped on purpose, they basically test nothing :-(